person.rb

```diff
@@ -1,30 +1,38 @@
+require_relative 'nameable'
+require_relative 'decorator'
+require_relative 'capitalize_decorator'
+require_relative 'trimmer_decorator'
+
+# Class representing a person
 class Person
```

Comment:
- [ ] Great job working on this milestone. As per project requirements, `Person` class should inherit from `Nameable`.
`class Person < Nameable`
![Screen Shot 2023-08-31 at 4 14 38 AM](https://github.com/Alibaba2023/oop-school/assets/62849872/479332cf-b27c-4be1-a881-ffe36f901a44)